Makefile

```diff
@@ -1,22 +1,21 @@
 # spaceteam Makefile
 
 $CXX = g++
-CXXFLAGS = -std=c++11 -Wall -Werror -isystem include
-BOOST_FLAGS = -lboost_system -lboost_filesystem
+CXXFLAGS = -std=c++11 -Wall -Werror -lpthread -isystem include
+BOOST_FLAGS = -lboost_system -lboost_filesystem -lpthread
 GTEST_DIR = googletest/googletest
 GMOCK_DIR = googletest/googlemock
-TEST_FLAGS = -std=c++11 -pthread
+TEST_FLAGS = -std=c++11 -lpthread
```

Comment:
On research, -lpthread seems to be the standard, and is considered a safer flag, but this is a revertible change, if you feel that -pthread satisfies your needs, this can be changed back